Jellyfin.Api/Controllers/SyncPlayController.cs

```diff
@@ -92,31 +97,137 @@ public ActionResult SyncPlayLeaveGroup()
         /// <returns>A <see cref="NoContentResult"/> indicating success.</returns>
         [HttpPost("Play")]
         [ProducesResponseType(StatusCodes.Status204NoContent)]
-        public ActionResult SyncPlayPlay()
+        public ActionResult SyncPlayPlay(
+            [FromQuery, Required] string playingQueue,
+            [FromQuery, Required] int playingItemPosition,
+            [FromQuery, Required] long startPositionTicks)
         {
             var currentSession = RequestHelpers.GetSession(_sessionManager, _authorizationContext, Request);
-            var syncPlayRequest = new PlaybackRequest()
+            var syncPlayRequest = new PlayGroupRequest()
             {
-                Type = PlaybackRequestType.Play
+                PlayingItemPosition = playingItemPosition,
+                StartPositionTicks = startPositionTicks
             };
+            syncPlayRequest.PlayingQueue.AddRange(RequestHelpers.GetGuids(playingQueue));
+            _syncPlayManager.HandleRequest(currentSession, syncPlayRequest, CancellationToken.None);
+            return NoContent();
+        }
+
+        /// <summary>
+        /// Request to change playlist item in SyncPlay group.
+        /// </summary>
+        /// <param name="playlistItemId">The playlist id of the item.</param>
+        /// <response code="204">Queue update request sent to all group members.</response>
+        /// <returns>A <see cref="NoContentResult"/> indicating success.</returns>
+        [HttpPost("SetPlaylistItem")]
+        [ProducesResponseType(StatusCodes.Status204NoContent)]
+        public ActionResult SyncPlaySetPlaylistItem(
+            [FromQuery, Required] string playlistItemId)
+        {
+            var currentSession = RequestHelpers.GetSession(_sessionManager, _authorizationContext, Request);
+            var syncPlayRequest = new SetPlaylistItemGroupRequest()
+            {
+                PlaylistItemId = playlistItemId
+            };
+            _syncPlayManager.HandleRequest(currentSession, syncPlayRequest, CancellationToken.None);
+            return NoContent();
+        }
+
+        /// <summary>
+        /// Request to remove items from the playlist in SyncPlay group.
+        /// </summary>
+        /// <param name="playlistItemIds">The playlist ids of the items to remove.</param>
+        /// <response code="204">Queue update request sent to all group members.</response>
+        /// <returns>A <see cref="NoContentResult"/> indicating success.</returns>
+        [HttpPost("RemoveFromPlaylist")]
+        [ProducesResponseType(StatusCodes.Status204NoContent)]
+        public ActionResult SyncPlayRemoveFromPlaylist(
+            [FromQuery, Required] string[] playlistItemIds)
+        {
+            var currentSession = RequestHelpers.GetSession(_sessionManager, _authorizationContext, Request);
+            var syncPlayRequest = new RemoveFromPlaylistGroupRequest();
```

Comment:
Pass playlistItemIds in through constructor, make `PlaylistItemIds` `IReadOnlyList<string>`